app/views/referee_interface/reference/feedback.html.erb

```diff
@@ -7,37 +7,35 @@
   <%= f.govuk_error_summary %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">Give a teacher training reference for <%= @application.full_name %></h1>
-
-      <p class="govuk-body"><%= @application.full_name %> applied to the following courses:</p>
+      <h1 class="govuk-heading-xl">Your reference for <%= @application.full_name %></h1>
 
+      <p class="govuk-body">When giving your reference try to cover the following, but don’t worry if there are areas you can’t comment on:</p>
       <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
-        <% @application.application_choices.each do |application_choice| %>
-          <li><%= application_choice.course.name %> - <%= application_choice.site.name %></li>
+        <% if @reference_form.reference.referee_type.nil? || @reference_form.reference.referee_type == 'academic' %>
+          <li>
+            <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Academic abilities</h2>
+            <p class="govuk-body">Does the candidate think critically and have good numeracy and literacy skills?</p>
+          </li>
```

Comment:
Regarding your question: Yeah, it does make sense to show this if there a reference doesn't have a referee type. I think system spec would be overkill for this. The alternative would be to create a component for this and then we can have coverage for this.

Reply:
I wrapped it into a component as you suggested 🎉👌